bitnami/minio/templates/servicemonitor.yaml

```diff
@@ -2,7 +2,7 @@
 Copyright Broadcom, Inc. All Rights Reserved.
 SPDX-License-Identifier: APACHE-2.0
 */}}
-
+{{- /* TODO: On major version upgrade, add .Values.metrics.enabled to that check */}}
```

Comment:
Given the default values is `false`, there shouldn't be any problem on adding the parameter on a minor version? I mean, there shouldn't be any upgradability issue, right?

Reply:
The people that are running with `metrics.serviceMonitor.enabled=true` but don't use the `metrics.prometheusRule` should have had no reason (at least no documented reason) to set `metrics.enabled=true`, so we would be removing the service monitor. This sounds like a breaking change to me but maybe I see it the wrong way

Reply:
Fair enough! Let's keep the TODO then.